$NetBSD$

- Don't use CMAKE_MODULE_PATH directly so pkgsrc can append it with
  multiple paths.
- Install freedesktop.org files unconditionally.

--- CMakeLists.txt.orig	2020-07-27 12:01:52.000000000 +0000
+++ CMakeLists.txt
@@ -38,8 +38,8 @@ include(ConkyPlatformChecks)
 include(ConkyCPackSetup)
 
 # setup our configuration headers
-configure_file(${CMAKE_MODULE_PATH}/config.h.in ${CMAKE_BINARY_DIR}/config.h)
-configure_file(${CMAKE_MODULE_PATH}/build.h.in ${CMAKE_BINARY_DIR}/build.h)
+configure_file(${CMAKE_SOURCE_DIR}/cmake/config.h.in ${CMAKE_BINARY_DIR}/config.h)
+configure_file(${CMAKE_SOURCE_DIR}/cmake/build.h.in ${CMAKE_BINARY_DIR}/build.h)
 
 set(conky_sources ${CMAKE_BINARY_DIR}/config.h ${CMAKE_BINARY_DIR}/build.h)
 
@@ -94,11 +94,9 @@ if(BUILD_DOCS)
   install(FILES ${MAN_FILES} DESTINATION ${MAN_PATH})
 endif(BUILD_DOCS)
 
-if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
   install(FILES conky.desktop DESTINATION share/applications)
   install(FILES logo/conky-logomark-violet.svg
           DESTINATION share/icons/hicolor/scalable/apps)
-endif(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
 
 if(CHECK_CODE_QUALITY)
   find_package(ClangTidy)
